staging/src/k8s.io/api/OWNERS

```diff
@@ -2,54 +2,9 @@
 options:
   no_parent_owners: true
 approvers:
-- erictune
-- lavalamp
```

Comment:
FYI I'm not officially in the api-approvers group so this may mean more approvals for the rest of you.